code/modules/surgery/tools.dm

```diff
@@ -115,6 +115,7 @@
 	flags_1 = CONDUCT_1
 	item_flags = SURGICAL_TOOL
 	force = 15
+	demolition_mod = 0.5
```

Comment:
idk I think a drill would be pretty good at tearing through a window or something

Reply:
Maybe a normal window, but we have super strong reinforced windows that I'd imagine you could drill a hole in and the structural integrity would not be impacted very much.